packages/react/src/combobox/Combobox.test.tsx

```diff
@@ -82,4 +82,14 @@ describe('Combobox', () => {
 		expect(inputRef.current).toBeInstanceOf(HTMLInputElement);
 		expect(inputRef.current?.id).toBe(id);
 	});
+
+	it('listBox is described by the label correctly', async () => {
```

Comment:
If this test was run on the `main` branch, it would fail. Adding this test case so we don't accidentally break this in the future.